cpp/bridges_module/bridges_module.cpp

```diff
@@ -0,0 +1,61 @@
+#include <mg_procedure.h>
+#include <mg_exceptions.hpp>
+#include <mg_utils.hpp>
+
+#include "algorithm/bridges.hpp"
+
+namespace {
+
+// const char *fieldEdgeID = "edge_id";
+const char *fieldNodeFrom = "node_from";
+const char *fieldNodeTo = "node_to";
```

Comment:
`kFieldNodeFrom` and `kFieldNodeTo`? Not sure what is used for constants in this code but it's good to have convention for that.